diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.18.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/emailsettings mezzanine_patched_e-smith-email-4.18.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/emailsettings
--- e-smith-email-4.18.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/emailsettings	2008-03-11 18:03:25.000000000 +0100
+++ mezzanine_patched_e-smith-email-4.18.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/emailsettings	2008-04-27 10:48:43.000000000 +0200
@@ -174,14 +174,6 @@
         <trans>Allow HTTPS (secure)</trans>
     </entry>
     <entry>
-        <base>PUBLIC</base>
-        <trans>Allow public access (entire Internet)</trans>
-    </entry>
-    <entry>
-        <base>PRIVATE</base>
-        <trans>Allow access only from local networks</trans>
-    </entry>
-    <entry>
         <base>INSECURE_POP3</base>
         <trans>Allow both POP3 and POP3S</trans>
     </entry>
@@ -319,10 +311,6 @@
         <base>LABEL_WEBMAIL</base>
         <trans>Webmail access</trans>
     </entry>
-    <entry>
-        <base>ERROR_UPDATING</base>
-        <trans>Error occurred while updating system configuration.</trans>
-    </entry>
      <entry>
         <base>LABEL_BLOCK_EXECUTABLE_CONTENT</base>
         <trans>Executable content blocking</trans>
diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.18.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/emailsettings.pm mezzanine_patched_e-smith-email-4.18.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/emailsettings.pm
--- e-smith-email-4.18.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/emailsettings.pm	2008-03-11 18:03:25.000000000 +0100
+++ mezzanine_patched_e-smith-email-4.18.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/emailsettings.pm	2008-04-27 10:47:14.000000000 +0200
@@ -243,7 +243,7 @@
 
     unless ( system( "/sbin/e-smith/signal-event", "email-update" ) == 0 )
     {
-	$fm->error('ERROR_UPDATING');
+	$fm->error('ERROR_UPDATING_CONFIGURATION');
 	return undef;
     }
     $fm->success('SUCCESS');
@@ -272,7 +272,7 @@
 
     unless ( system( "/sbin/e-smith/signal-event", "email-update" ) == 0 )
     {
-	$fm->error('ERROR_UPDATING');
+	$fm->error('ERROR_UPDATING_CONFIGURATION');
 	return undef;
     }
     $fm->success('SUCCESS');
@@ -352,7 +352,7 @@
     
     unless ( system( "/sbin/e-smith/signal-event", "email-update" ) == 0 )
     {
-	$fm->error('ERROR_UPDATING');
+	$fm->error('ERROR_UPDATING_CONFIGURATION');
 	return undef;
     }
 
@@ -383,7 +383,7 @@
 
     unless ( system( "/sbin/e-smith/signal-event", "email-update" ) == 0 )
     {
-	$fm->error('ERROR_UPDATING');
+	$fm->error('ERROR_UPDATING_CONFIGURATION');
 	return undef;
     }
 
@@ -708,7 +708,7 @@
 {
     my %options = (
 	disabled => 'DISABLED',
-	private => 'ALLOW_PRIVATE',
+	private => 'ALLOW_NETWORKS_ALLOW_LOCAL',
 	publicSSL  => 'SECURE_POP3'
     );
 
@@ -723,7 +723,7 @@
 {
     my %options = (
 	disabled => 'DISABLED',
-	private => 'ALLOW_PRIVATE',
+	private => 'ALLOW_NETWORKS_ALLOW_LOCAL',
 	publicSSL  => 'SECURE_IMAP'
     );
 
