diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.18.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/pseudonyms.pm mezzanine_patched_e-smith-email-4.18.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/pseudonyms.pm
--- e-smith-email-4.18.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/pseudonyms.pm	2008-03-11 17:03:25.000000000 +0000
+++ mezzanine_patched_e-smith-email-4.18.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/pseudonyms.pm	2008-07-26 17:37:22.000000000 +0100
@@ -164,7 +164,7 @@
     my $fm = shift;
     my $q = $fm->{'cgi'};
     my $account = $q->param ('account');
-    my $pseudonym = $q->param('pseudonym');
+    my $pseudonym = uri_unescape($q->param('pseudonym')); 
     my $msg = "OK";
 
     $accounts->new_record($pseudonym, { type => 'pseudonym', 
@@ -355,7 +355,7 @@
         my $key = $account->key();
         push (@emailPseudonyms, $key) if ($type eq 'pseudonym');
     }
-    my $urlprefix = $fm->build_cgi_params($q->param('pseudonym'));    
+    my $urlprefix = $fm->build_cgi_params(uri_unescape($q->param('pseudonym')));    
     print "  <tr>\n    <td colspan='2'>\n";
     print $q->p($q->a({href => "pseudonyms?$urlprefix&wherenext=Create", -class => "button-like"},
                 $fm->localise("CLICK_TO_CREATE")));
@@ -419,7 +419,7 @@
 sub get_pseudonym_account {
     my $fm = shift;
     my $q = $fm->{'cgi'};
-    my $pseudonym = $q->param('pseudonym');
+    my $pseudonym = uri_unescape($q->param('pseudonym'));
     my $account = $accounts->get($pseudonym)->prop('Account');
     if ($account eq "admin") {
         $account = "Administrator";
@@ -441,7 +441,7 @@
 sub is_pseudonym_not_removable {
     my $fm = shift;
     my $q = $fm->{'cgi'};
-    my $pseudonym = $q->param('pseudonym');
+    my $pseudonym = uri_unescape($q->param('pseudonym'));
     my $removable = $accounts->get($pseudonym)->prop('Removable') || 'yes';
     return 1  if ($removable eq 'yes');
     return 0;
@@ -459,7 +459,7 @@
 sub is_pseudonym_internal {
     my $fm = shift;
     my $q = $fm->{'cgi'};
-    my $pseudonym = $q->param('pseudonym');
+    my $pseudonym = uri_unescape($q->param('pseudonym'));
     my $visible = $accounts->get($pseudonym)->prop('Visible') || '';
     return 'YES'  if ($visible eq 'internal');
     return 'NO';
