--- e-smith-email-4.16.0/root/etc/e-smith/web/functions/emailsettings.spamreject	2007-06-12 11:57:09.000000000 -0600
+++ e-smith-email-4.16.0/root/etc/e-smith/web/functions/emailsettings	2007-06-12 11:58:03.000000000 -0600
@@ -419,7 +419,7 @@
             id="SpamRejectLevel"
 	    display="get_prop('spamassassin', 'type')"
 	    options="get_spam_level_options()"
-            value="get_prop('spamassassin','RejectLevel')">
+            value="get_prop('spamassassin','RejectLevel','0')">
             <label>LABEL_SPAM_REJECTLEVEL</label>
 	</field>
 
