diff -up e-smith-email-5.2.0/root/usr/local/sbin/smtp-auth-proxy.pl.HeloHost e-smith-email-5.2.0/root/usr/local/sbin/smtp-auth-proxy.pl
--- e-smith-email-5.2.0/root/usr/local/sbin/smtp-auth-proxy.pl.HeloHost	2008-03-11 18:03:25.000000000 +0100
+++ e-smith-email-5.2.0/root/usr/local/sbin/smtp-auth-proxy.pl	2009-12-21 21:57:02.000000000 +0100
@@ -29,6 +29,8 @@ sub options {
     $prop->{SystemName} = $config->get_value('SystemName');
     $prop->{DomainName} = $config->get_value('DomainName');
     $prop->{PeerPort} = $smtp_proxy_rec->prop('PeerPort') || 25;
+    $prop->{Helo} = $config->get('smtpd')->prop('HeloHost') ||
+        "$prop->{SystemName}.$prop->{DomainName}";
 }
 
 esmith::SMTPAuthProxy->run(
@@ -52,6 +54,7 @@ sub process_request
     my $user = $self->get_property('user');
     my $domain_name = $self->get_property('DomainName');
     my $system_name = $self->get_property('SystemName');
+    my $helo = $self->get_property('Helo');
     my $name = $self->get_property('Userid');
     my $pass = $self->get_property('Passwd');
     my $debug = (($self->get_property('Debug') || 'disabled') eq 'enabled')
@@ -74,7 +77,7 @@ sub process_request
 
     my $class = ($port == 465) ? 'Net::SMTP::SSL' : 'Net::SMTP';
     my $smtp = $class->new($smarthost,
-                            Hello => "${system_name}.${domain_name}",
+                            Hello => Hello => $helo,
                             Debug => $debug,
                             Port => $port,
                         );
