diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.16.0/root/etc/e-smith/events/actions/purge-junkmail-folders mezzanine_patched_e-smith-email-4.16.0/root/etc/e-smith/events/actions/purge-junkmail-folders
--- e-smith-email-4.16.0/root/etc/e-smith/events/actions/purge-junkmail-folders	2006-03-15 06:23:58.000000000 +1100
+++ mezzanine_patched_e-smith-email-4.16.0/root/etc/e-smith/events/actions/purge-junkmail-folders	2006-09-27 06:50:14.000000000 +1000
@@ -31,7 +31,8 @@
     my $pwent = getpwnam($user) 
 	or die "Couldn't get password entry for $user\n";
 
-    chdir $pwent->dir or die "Couldn't chdir $pwent->dir:$!\n";
+    chdir $pwent->dir or die "Couldn't chdir " . $pwent->dir . ":$!\n";
+    
     unless (chdir "Maildir/.junkmail")
     {
 	warn("Couldn't chdir to junkmail folder of user $user: $!\n");
