diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3/access mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3/access
--- e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3/access	2005-08-11 04:55:15.000000000 +1000
+++ mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3/access	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-private
diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3/status mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3/status
--- e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3/status	2005-08-11 04:55:15.000000000 +1000
+++ mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3/status	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-enabled
diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3/TCPPort mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3/TCPPort
--- e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3/TCPPort	2005-08-11 04:55:15.000000000 +1000
+++ mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3/TCPPort	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-110
diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3/type mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3/type
--- e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3/type	2005-08-11 04:55:15.000000000 +1000
+++ mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3/type	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-service
diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3s/access mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3s/access
--- e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3s/access	2005-03-10 12:41:09.000000000 +1100
+++ mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3s/access	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-private
diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3s/status mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3s/status
--- e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3s/status	2005-03-10 12:41:09.000000000 +1100
+++ mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3s/status	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-enabled
diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3s/TCPPort mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3s/TCPPort
--- e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3s/TCPPort	2005-03-10 12:41:09.000000000 +1100
+++ mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3s/TCPPort	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-995
diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3s/type mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3s/type
--- e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3s/type	2005-03-10 12:41:09.000000000 +1100
+++ mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/db/configuration/defaults/pop3s/type	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-service
