diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.15.4/createlinks mezzanine_patched_e-smith-email-4.15.4/createlinks
--- e-smith-email-4.15.4/createlinks	2005-08-11 05:16:23.000000000 +1000
+++ mezzanine_patched_e-smith-email-4.15.4/createlinks	2006-01-07 00:43:51.174818892 +1100
@@ -51,14 +51,6 @@
 safe_symlink("restart", "root/etc/e-smith/events/$event/services2adjust/smtp-auth-proxy");
 
 #--------------------------------------------------
-# actions for bootstrap-console-save event:
-# write config files and create startup link
-#--------------------------------------------------
-$event = "bootstrap-console-save";
-
-event_link("update-user-pseudonyms", $event, "25");
-
-#--------------------------------------------------
 # actions for domain-create event:
 # rewrite config files and restart server
 #--------------------------------------------------
