diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.15.4/root/etc/e-smith/web/functions/emailsettings mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/web/functions/emailsettings
--- e-smith-email-4.15.4/root/etc/e-smith/web/functions/emailsettings	2006-02-02 04:58:11.083514379 +1100
+++ mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/web/functions/emailsettings	2006-02-02 04:53:34.494637735 +1100
@@ -511,7 +511,7 @@
         </field>
 
         <field 
-            type="text" 
+            type="password" 
             id="SMTPAUTHPROXY_Passwd" 
             validation="nonblank_if_smtpauth()" 
             value="get_prop('smtp-auth-proxy', 'Passwd')">
