diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.15.4/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/pseudonyms mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/pseudonyms
--- e-smith-email-4.15.4/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/pseudonyms	2005-05-02 02:38:54.000000000 +1000
+++ mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/pseudonyms	2006-02-02 05:59:06.000000000 +1100
@@ -39,7 +39,22 @@
         on the system and a pseudonym "everyone" which contains
         all users on the system.</p>
 
-        <p>You can modify or remove a
+    <p>Pseudonyms also allow you to create e-mail aliases for valid
+	(virtual) domains & users. For example you can forward
+	"sales@virtualdomain1.com" to user "mary" and 
+	"sales@virtualdomain2.com" to user "john".
+	You must create the domains and user accounts before creating
+	the email aliases.</p>
+
+    <p>Pseudonyms can also forward to other pseudonyms, although this
+	is limited to one further level. For example the pseudonym
+	"webmaster" can forward to the pseudonym "techsupport" which
+	then forwards to a specified user "ray". This is useful where
+	one user is responsible	for multiple roles and saves having
+	to change the pseudonym's associated with that user many times
+	in the event that the user departs the organisation.</p>
+
+    <p>You can modify or remove a
         pseudonym by clicking on the corresponding
         command next to the pseudonym.</p>]]>
     </trans>
@@ -106,6 +121,11 @@
            number. For example "sales", "john.holland", "123" 
            and "email-administrator" are all valid pseudonyms, 
            but "John Smith" and "Henry Miller" are not.
+	
+	<p>You can also create email aliases for accounts and (virtual)
+	domains that are valid on this server. For example
+	"fred@virtualdomain.com" can be forwarded to "mary"
+	(or to another valid user account).</p>
            ]]>
         </trans>
     </entry>
