diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.16.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/emailsettings mezzanine_patched_e-smith-email-4.16.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/emailsettings
--- e-smith-email-4.16.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/emailsettings	2006-09-20 08:06:59.284128056 +1000
+++ mezzanine_patched_e-smith-email-4.16.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/emailsettings	2006-09-20 07:59:22.260729262 +1000
@@ -485,8 +485,8 @@
     <entry>
 	<base>DESC_VIRUS_SCAN</base>
 	<trans>
-		You can scan e-mail for viruses and either reject 
-		or quarantine messages containing viruses.
+		You can scan incoming and outgoing e-mail for viruses. If scanning is enabled and a virus is detected, the e-mail will be rejected and returned to the
+sender.
 	</trans>
     </entry>
 
