diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.15.4/root/etc/e-smith/templates.metadata/home/e-smith/.qmail mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/templates.metadata/home/e-smith/.qmail
--- e-smith-email-4.15.4/root/etc/e-smith/templates.metadata/home/e-smith/.qmail	2005-01-17 17:33:05.000000000 -0500
+++ mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/templates.metadata/home/e-smith/.qmail	1969-12-31 19:00:00.000000000 -0500
@@ -1,3 +0,0 @@
-UID="admin"
-GID="admin"
-PERMS=0640
diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.15.4/root/etc/e-smith/templates-user/.qmail-junkmail/10deliver_to_junkmail mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/templates-user/.qmail-junkmail/10deliver_to_junkmail
--- e-smith-email-4.15.4/root/etc/e-smith/templates-user/.qmail-junkmail/10deliver_to_junkmail	2005-05-13 15:10:11.000000000 -0400
+++ mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/templates-user/.qmail-junkmail/10deliver_to_junkmail	1969-12-31 19:00:00.000000000 -0500
@@ -1 +0,0 @@
-./Maildir/.junkmail/
