diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.15.4/root/etc/e-smith/db/configuration/migrate/35imp mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/db/configuration/migrate/35imp
--- e-smith-email-4.15.4/root/etc/e-smith/db/configuration/migrate/35imp	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/db/configuration/migrate/35imp	2006-02-04 23:45:24.328831462 +1100
@@ -0,0 +1,8 @@
+{
+    my $imp = $DB->get('imp') or return;
+    
+    return unless ($imp->prop('access') eq 'full');
+    
+    $imp->set_prop('access','SSL');
+    
+}
\ No newline at end of file
