diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/Body mezzanine_patched_e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/Body
--- e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/Body	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/Body	2006-06-21 19:45:41.936124060 +1000
@@ -0,0 +1 @@
+TVp1AQEAAAAE
diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/Description mezzanine_patched_e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/Description
--- e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/Description	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/Description	2006-06-21 19:45:49.086883174 +1000
@@ -0,0 +1 @@
+PC executables (TVp1AQEAAAAE)
diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/Glob mezzanine_patched_e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/Glob
--- e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/Glob	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/Glob	2006-06-21 19:45:22.849436351 +1000
@@ -0,0 +1 @@
+yes
diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/LineStart mezzanine_patched_e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/LineStart
--- e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/LineStart	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/LineStart	2006-06-21 19:45:22.849436351 +1000
@@ -0,0 +1 @@
+yes
diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/Status mezzanine_patched_e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/Status
--- e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/Status	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/Status	2006-06-21 19:45:22.849436351 +1000
@@ -0,0 +1 @@
+disabled
diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/type mezzanine_patched_e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/type
--- e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/type	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-email-4.16.0/root/etc/e-smith/db/mailpatterns/defaults/TVp1AQEAAAAE/type	2006-06-21 19:45:55.062846174 +1000
@@ -0,0 +1 @@
+pattern
