diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.16.0/root/etc/e-smith/db/configuration/migrate/35imp mezzanine_patched_e-smith-email-4.16.0/root/etc/e-smith/db/configuration/migrate/35imp
--- e-smith-email-4.16.0/root/etc/e-smith/db/configuration/migrate/35imp	2006-03-15 06:23:58.000000000 +1100
+++ mezzanine_patched_e-smith-email-4.16.0/root/etc/e-smith/db/configuration/migrate/35imp	1970-01-01 10:00:00.000000000 +1000
@@ -1,8 +0,0 @@
-{
-    my $imp = $DB->get('imp') or return;
-    
-    return unless ($imp->prop('access') eq 'full');
-    
-    $imp->set_prop('access','SSL');
-    
-}
