diff -Nur -x '*.orig' -x '*.rej' e-smith-email-4.15.4/root/etc/e-smith/web/functions/emailsettings mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/web/functions/emailsettings
--- e-smith-email-4.15.4/root/etc/e-smith/web/functions/emailsettings	2005-09-14 00:34:15.000000000 +1000
+++ mezzanine_patched_e-smith-email-4.15.4/root/etc/e-smith/web/functions/emailsettings	2006-01-25 13:41:10.427788875 +1100
@@ -177,6 +177,7 @@
         <field 
             type="literal" 
             id="state_WebMail" 
+	    display="get_prop('imp', 'type')"
             value="get_current_webmail_status('localise')">
             <label>LABEL_WEBMAIL</label>
         </field>
@@ -191,6 +192,7 @@
         <field 
             type="literal"
             id="state_Virus"
+	    display="get_prop('clamav', 'type')"
             value="get_virus_status('localise')">
             <label>LABEL_VIRUS_SCAN</label>
             <description>DESC_SECTIONBAR</description>
@@ -199,6 +201,7 @@
         <field 
             type="literal"
             id="state_Spam"
+	    display="get_prop('spamassassin', 'type')"
             value="get_spam_status('localise')">
             <label>LABEL_SPAM_SCAN</label>
 	</field>
@@ -375,6 +378,7 @@
 
         <field type="select"
             id="VirusStatus"
+	    display="get_prop('clamav', 'type')"
             options="'enabled' => 'ENABLED', 'disabled' => 'DISABLED'"
             value="get_virus_status()">
             <label>LABEL_VIRUS_SCAN</label>
@@ -383,6 +387,7 @@
 
         <field type="select"
             id="Spamstatus"
+	    display="get_prop('spamassassin', 'type')"
             options="'enabled' => 'ENABLED', 'disabled' => 'DISABLED'"
             value="get_spam_status()">
             <label>LABEL_SPAM_SCAN</label>
@@ -391,6 +396,7 @@
 
         <field type="select"
             id="SpamSensitivity"
+	    display="get_prop('spamassassin', 'type')"
  	    options=" 'verylow' => 'VERYLOW', 'low'=>'LOW', 'medium'=>'MEDIUM', 'high'=>'HIGH', 'veryhigh' => 'VERYHIGH', 'custom' => 'CUSTOM' "
             value="get_prop('spamassassin','Sensitivity','medium')">
             <label>LABEL_SPAM_SENSITIVITY</label>
@@ -399,6 +405,7 @@
         <field
             type="select"
             id="SpamTagLevel"
+	    display="get_prop('spamassassin', 'type')"
 	    options="get_spam_level_options()"
             value="get_prop('spamassassin','TagLevel')">
             <label>LABEL_SPAM_TAGLEVEL</label>
@@ -407,6 +414,7 @@
         <field
             type="select"
             id="SpamRejectLevel"
+	    display="get_prop('spamassassin', 'type')"
 	    options="get_spam_level_options()"
             value="get_prop('spamassassin','RejectLevel')">
             <label>LABEL_SPAM_REJECTLEVEL</label>
@@ -415,6 +423,7 @@
         <field
             type="select"
             id="SpamSortSpam"
+	    display="get_prop('spamassassin', 'type')"
             options=" 'enabled'=>'ENABLED', 'disabled'=>'DISABLED' "
             value="get_prop('spamassassin','SortSpam','enabled')">
             <label>LABEL_SORTSPAM</label>
@@ -423,6 +432,7 @@
         <field
             type="select"
             id="SpamSubjectTag"
+	    display="get_prop('spamassassin', 'type')"
             options=" 'enabled'=>'ENABLED', 'disabled'=>'DISABLED' "
             value="get_prop('spamassassin','SubjectTag')">
             <label>LABEL_SPAM_SUBJECTTAG</label>
@@ -530,6 +540,7 @@
         <field 
             type="select" 
             id="WebMail" 
+	    display="get_prop('imp', 'type')"
             options="get_webmail_options()"
             value="get_current_webmail_status()">
             <label>LABEL_WEBMAIL</label>
